src/config.cpp

```diff
@@ -1,29 +1,28 @@
-/*
-obs-ndi
-Copyright (C) 2016-2023 Stéphane Lepin <stephane.lepin@gmail.com>
+/******************************************************************************
+	Copyright (C) 2016-2024 DistroAV <distroav@distroav.org>
 
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation; either version 2 of the License, or
-(at your option) any later version.
+	This program is free software; you can redistribute it and/or
+	modify it under the terms of the GNU General Public License
+	as published by the Free Software Foundation; either version 2
+	of the License, or (at your option) any later version.
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
 
-You should have received a copy of the GNU General Public License along
-with this program. If not, see <https://www.gnu.org/licenses/>
-*/
+	You should have received a copy of the GNU General Public License
+	along with this program; if not, see <https://www.gnu.org/licenses/>.
+******************************************************************************/
```

Comment:
@paulpv Any reason for the fancy tab/blank-spaces at the line beginning?

(I personally do not mind the extra ***)

Reply:
I copied the tabbed look from OBS.
They are very scattered:  
https://github.com/search?q=repo%3Aobsproject%2Fobs-studio%20Copyright&type=code

I liked the margined look of the tabbed version best.  

Since I have to touch them all anyway, I just took the time to make them all consistent and look a little prettier to me.

I also refreshed the text from https://www.gnu.org/licenses/old-licenses/gpl-2.0.html#SEC4 . The only change was they provide a snail-mail address that I kept as a url.

Reply:
Resolving this